plugin/src/main/java/at/helpch/chatchat/channel/AbstractChannel.java

```diff
@@ -57,9 +57,12 @@ public int radius() {
         return radius;
     }
 
-
     @Override
-    public boolean isUseableBy(@NotNull final ChatUser user) {
+    public boolean isUsableBy(@NotNull final ChatUser user) {
+        if (name.equals("default")) {
```

Comment:
The default channel will not always be named default. Users can chose to change their default channel from the channels.yml. A better way to do this would be something like this:
```java
ChatChannel.defaultChannel().equals(this)
```